### ISSUE TITLE
Remove CPU architecture detection for Linux

### ISSUE THREAD

**Issue by alexgibson** [mozilla/bedrock#14012]
## Description

See https://bugzilla.mozilla.org/show_bug.cgi?id=1861847

The TL:DR is Firefox will soon freeze the Linux user agent string header (as well as `navigator.userAgent` and `navigator.platform`) to always include `Linux x86_64`. This will break our Linux downloads, serving 64-bit binaries to visitors on 32-bit systems.

Because Firefox does not also support Client Hints API, detecting CPU architecture on Linux will no longer be possible for serving the correct file download.

We need to remove our reliance on this for Linux, and offer people a choice of which file to download instead.

See also https://github.com/mozilla/bedrock/issues/12966 for related upcoming changes on Linux.

---

#### Success Criteria
- [ ] CPU architecture detection for serving Firefox downloads on Linux is removed.
- [ ] Linux visitors can still find a pathway to download either 32bit of 64bit binaries.
